README_ZH.md

```diff
@@ -48,33 +48,23 @@ See more: https://go-proverbs.github.io/
 - 每个 package 实现单一的目的
 - 显式处理错误
 - 尽早返回，而不是使用深嵌套
-- 让调用者选择并发
+- 让调用者处理并发（带来的问题）
```

Comment:
What is inside `(..)` ?

Reply:
some assistant phrases for comprehension

Reply:
that's to say, leave concurrency (and it's consequence) to caller.